perf/benchmark/runner/runner.py

```diff
@@ -269,10 +272,10 @@ def generate_nighthawk_cmd(self, cpus, conn, qps, duration, labels):
             "--output-format json",
             "--prefetch-connections",
             "--open-loop",
+            "--jitter-uniform 0.0001s",
```

Comment:
@oschaaf what's the value should we set here in order to compare with fortio? Since as far as i know, fortio only has flag: `-jitter` set to be True of False. Do not need to specify a value for it.

Reply:
At 1000 qps, the best approximation of Fortio's `--jitter` feature in NH is `--jitter-uniform 0.0001s`.
The computation for that value looks like 1/qps/10, so it would be more robust to run that computation
instead of hard-code the timing, to more faithfully emulate.

Rationale:
Fortio's jitter computation is implemented as:

```go

targetElapsedDuration := time.Duration(int64(targetElapsedInSec * 1e9))
sleepDuration := targetElapsedDuration - elapsed
if r.Jitter {
  sleepDuration += getJitter(sleepDuration)
}

...

// getJitter returns a jitter time that is (+/-)10% of the duration t if t is >0
func getJitter(t time.Duration) time.Duration {
	if t <= 0 {
		return time.Duration(0)
	}
	i := int64(t / 10)
	j := rand.Int63n(2*i) - i
	return time.Duration(j)
}
```

If I read this correctly, it evenly distributes request-release timings around 1/10th fraction of the approximate (I suspect this might get noisy / diverge when rps starts lagging qps and/or sleep timings get noisy) request-release period on both sides of the unjittered timings.